README.md

```diff
@@ -15,7 +15,7 @@ This project contains Terraform scripts to provision the AWS cloud infrastructur
 [<img src="./docs/images/viya4-iac-aws-diag.png" alt="Architecture Diagram" width="750"/>](./docs/images/viya4-iac-aws-diag.png?raw=true)
 
 This project helps you to automate the cluster-provisioning phase of SAS Viya platform deployment. To learn about all phases and options of the
-SAS Viya platform deployment process, see [Getting Started with SAS Viya and AWS Elastic Kubernetes Service](https://go.documentation.sas.com/doc/en/itopscdc/v_037/itopscon/p111dd1k5z99kin1f7mciggn9wop.htm) in _SAS&reg; Viya&reg; Platform Operations_.
+SAS Viya platform deployment process, see [Getting Started with SAS Viya and AWS Elastic Kubernetes Service](https://go.documentation.sas.com/doc/en/itopscdc/default/itopscon/p111dd1k5z99kin1f7mciggn9wop.htm) in _SAS&reg; Viya&reg; Platform Operations_.
```

Comment:
I think the term is Amazon Elastic Kubernetes Service.

Reply:
Updated 'AWS' to 'Amazon'.